src/utils/Global.js

```diff
@@ -125,5 +125,10 @@ define(function (require, exports, module) {
     // only be able to load modules that have already been loaded once.
     global.brackets.getModule = require;
 
+    // API for retrieving the global RequireJS config
+    global.brackets.getGlobalRequireJSConfig = function () {
```

Comment:
Are you going to expose this to extensions? Otherwise can you prefix with _ and add a comment that is for internal use only?